src/containers/FaqAccordion.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  Container,
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography
+} from '@mui/material';
+import Iconify from '../components/Iconify';
+
+const ExpandIcon = () => (
+  <Iconify icon="material-symbols:expand-more-rounded" width={12} height={12} />
+);
+
+const faq = [
+  {
+    question: 'What is WIN.so?',
+    answer:
+      'Win.so is a web 2.5 accommodation booking platform created by Winding Tree and Simard teams. It operates on smart contract technology and uses crypto payments.'
+  },
+  {
+    question: 'What is Winding Tree?',
+    answers:
```

Comment:
This is the point I mentioned on call regarding loosing commits. This file was fixed in previous commit, should be `answer`